spec/models/user_spec.rb

```diff
@@ -54,4 +65,90 @@
       it { is_expected.not_to accept_values_for(:time_zone, nil, "Foo Bar") }
     end
   end
+
+  describe ".for_timer_reminder" do
+    let(:date) { Date.parse("Monday") }
+
+    before do
+      @user = create(:user, workdays: %w(1 2))
+      @other_user = create(:user, workdays: %w(4))
+      day = create(:day, date: date, user: @user, client_hours: 0,
+                   internal_hours: 0, pto: false,
+                   timer_reminder_sent: false, workday: true)
+    end
+
+    it "includes users that work that day and haven't started a timer" do
+      expect(User.for_timer_reminder(date: date)).to include(@user)
+    end
+
+    it "doesn't include other users" do
+      expect(User.for_timer_reminder(date: date)).not_to include(@other_user)
+    end
+  end
+
+  describe "#timer_reminder_sent!" do
+    before do
+      @user = create(:user)
+      @tomorrow_day = create(:day, date: Date.tomorrow, user: @user)
+      @today_day = create(:day, date: Date.current, user: @user)
+    end
+
+    it "sets the day's timer_reminder_sent to true when date passed" do
+      @user.timer_reminder_sent!(date: Date.tomorrow)
+      @tomorrow_day.reload
+
+      expect(@tomorrow_day.timer_reminder_sent).to eq(true)
+    end
+
+    it "sets today's timer_reminder_sent to true when no date passed" do
+      @user.timer_reminder_sent!
+      @today_day.reload
+
+      expect(@today_day.timer_reminder_sent).to eq(true)
+    end
```

Comment:
I started off trying to use ```let```s, but because it builds a SQL string & runs an actual query, it doesn't work that way. At least, I couldn't get it to work that way.